docs/topics/mta-web-subv-rep.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-subv-rep_{context}"]
+= Configuring Subversion repositories
+
+
+.Procedure
+
+. In the left menu of *Administrator* view, click *Repositories* and then *Subversion*.
+. Click, *Consume insecure Git repositories* toggle switch to enable.
```

Comment:
I think same content is mentioned here:
https://github.com/windup/windup-documentation/pull/572/files#diff-4afe5b22018a43d3e8bf8012ac5186e81dd02b21b6b1ab9fb953eb67555438fdR15
Can you please check if there is no duplication for repositories configuration altogether?

Reply:
There are 2 separate sub-categories that look similar, but are different. Richard is working on Configuring credentials and I am working on Configuring repositories. The headings look similar but they are different topics.